test/indexer.test.ts

```diff
@@ -88,4 +93,23 @@ describe('Indexer', () => {
     const key = `${Contract.Accounts}_${Event.AccountWalletAddressSet}`
     expect(await getLastBlock(key)).toEqual(0)
   })
+
+  it('getBlockTimestamps returns a mapping from blockNumber to timestamp', async () => {
+    const blockNumber1 = 1234
+    const blockNumber2 = 1235
+    const kit = await getContractKit()
+    const blockNumberToTimestamp = await getBlockTimestamps([partialEventLog({blockNumber: blockNumber1}), partialEventLog({blockNumber: blockNumber2})], kit)
+
+    expect(blockNumberToTimestamp).toEqual({[blockNumber1]: timestamp, [blockNumber2]: timestamp})
```

Comment:
this would be a stronger test if it checked two distinct timestamps. one easy way to accomplish this would be to change from hardcoding the response to using `jest.fn().mockImplementation((blockNumber) => blockNumber)` . The mock implementation doesn't give a "realistic" response as far as mapping block numbers to timestamps, but it at least returns 2 different outputs for 2 different inputs.

Reply:
Updated the tests to use this approach!